### PR TITLE
Fix some Japanese typos

### PR DESCRIPTION
**Pull request by kachick**
## Summary by CodeRabbit

- **Bug Fixes**
	- Corrected minor grammatical errors in markdown content to enhance clarity.

- **Documentation**
	- Improved the overall accuracy of the text recounting experiences related to game development and board games.
	- Updated link text in GitHub documentation reference for better clarity.

### CODE REVIEW

File: content/posts/my-back-is-dead.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ categories: ["腰"]
 
 ## 経緯
 
-- 2021-11-14 (日) に妻の親戚が家にやってきた。鬼滅の刃のボードゲームを持ってきてくれた。さっそくやろういうことで開封しつつ、コマやらカードやらの準備を進めた。このとき僕は「あぐら」の姿勢で作業をしていた。
+- 2021-11-14 (日) に妻の親戚が家にやってきた。鬼滅の刃のボードゲームを持ってきてくれた。さっそくやろうということで開封しつつ、コマやらカードやらの準備を進めた。このとき僕は「あぐら」の姿勢で作業をしていた。
```
Comment (kachick) [inline content/posts/my-back-is-dead.md:14]: こっちはありえる表現だけどなんか脱字っぽいなと思ったので直しておきました！

Comment (pankona) [reply content/posts/my-back-is-dead.md:14]: タイポやな！w

**Review by pankona — APPROVED** [review]
Azs！マージしていきます

**Comment by kachick** [comment]
あざざます！(not typo)

**Pull request merged**